include/d/a/d_a_item.h

```diff
@@ -44,8 +44,8 @@ class dAcItem_c : public dAcItemBase_c {
 
     static u32 createItemParams(u16 itemId, u32 subType, u32 unk, u8 unk2, u8, u8);
 
-    static void spawnItem(u16 item, u32 room, const mVec3_c &pos, const mAng3_c &rot, u32 params, u32 arg);
-    static void spawnDrop(u16 item, u32 room, const mVec3_c &pos, const mAng3_c &rot);
+    static void spawnItem(u16 item, u32 room, mVec3_c *pos, const mAng3_c &rot, u32 params, u32 arg);
+    static void spawnDrop(u16 item, u32 room, mVec3_c *pos, const mAng3_c &rot);
```

Comment:
what was the reasoning for this change? 
Its fine if required, but i think it would be good to keep both pos and rot the same format

Reply:
While doing stuff with `cLib::targetAngleY`, I wanted to make sure there was no regression on other stuff that used it. This squiggles were causing problems. Not sure why I changed that specifically tho.
![image](https://github.com/user-attachments/assets/4b39e3b7-6563-41c0-9f58-7a017c1260af)